### ISSUE TITLE
TASK [openshift_examples : Import Centos Image streams] Fails

### ISSUE THREAD

**Issue by nickkeyzer** [openshift/openshift-ansible#7129]
#### TASK [openshift_examples : Import Centos Image streams] Fails

Testing cluster upgrades using the Vagrant file included in the `openshift-ansible-contrib` project. Openshift Origin 3.6 deploys successfully but running the Openshift 3.7 upgrade fails.

##### Version

```
ansible 2.4.0.0

openshift-ansible-3.6.173.0.44-1 > openshift-ansible-3.7.29-1
```

##### Steps To Reproduce
1. Deploy Openshift 3.6 (`openshift-ansible-3.6.173.0.44-1`) cluster using Vagrantfile from `openshift-origin-contrib` project (https://github.com/openshift/openshift-ansible-contrib ).
2. Run cluster upgrade using `playbooks/byo/openshift-cluster/upgrades/v3_7/upgrade.yml`

##### Expected Results
Successful Ansible run.

##### Observed Results
Ansible fails on "_TASK [openshift_examples : Import Centos Image streams]_".  

```
TASK [openshift_examples : Import Centos Image streams] *************************************************************************************************************************************
failed: [master1] (item=/usr/share/openshift/examples/image-streams/image-streams-centos7.json) => {"changed": false, "cmd": ["oc", "replace", "--config=/etc/origin/master/admin.kubeconfig", "-n", "openshift", "-f", "/usr/share/openshift/examples/image-streams/image-streams-centos7.json"], "delta": "0:05:01.784549", "end": "2018-02-12 22:03:23.965998", "failed_when_result": true, "item": "/usr/share/openshift/examples/image-streams/image-streams-centos7.json", "msg": "non-zero return code", "rc": 1, "start": "2018-02-12 21:58:22.181449", "stderr": "Unable to connect to the server: net/http: TLS handshake timeout\nError from server (Invalid): imagestreams \"ruby\" is invalid: metadata.resourceVersion: Invalid value: 0x0: must be specified for an update", "stderr_lines": ["Unable to connect to the server: net/http: TLS handshake timeout", "Error from server (Invalid): imagestreams \"ruby\" is invalid: metadata.resourceVersion: Invalid value: 0x0: must be specified for an update"], "stdout": "imagestream \"nodejs\" replaced\nimagestream \"perl\" replaced\nimagestream \"php\" replaced\nimagestream \"python\" replaced\nimagestream \"wildfly\" replaced\nimagestream \"mysql\" replaced\nimagestream \"mariadb\" replaced\nimagestream \"postgresql\" replaced\nimagestream \"mongodb\" replaced\nimagestream \"redis\" replaced\nimagestream \"jenkins\" replaced", "stdout_lines": ["imagestream \"nodejs\" replaced", "imagestream \"perl\" replaced", "imagestream \"php\" replaced", "imagestream \"python\" replaced", "imagestream \"wildfly\" replaced", "imagestream \"mysql\" replaced", "imagestream \"mariadb\" replaced", "imagestream \"postgresql\" replaced", "imagestream \"mongodb\" replaced", "imagestream \"redis\" replaced", "imagestream \"jenkins\" replaced"]}
failed: [master1] (item=/usr/share/openshift/examples/image-streams/dotnet_imagestreams_centos.json) => {"changed": false, "cmd": ["oc", "replace", "--config=/etc/origin/master/admin.kubeconfig", "-n", "openshift", "-f", "/usr/share/openshift/examples/image-streams/dotnet_imagestreams_centos.json"], "delta": "0:04:17.811513", "end": "2018-02-12 22:07:43.334583", "failed_when_result": true, "item": "/usr/share/openshift/examples/image-streams/dotnet_imagestreams_centos.json", "msg": "non-zero return code", "rc": 1, "start": "2018-02-12 22:03:25.523070", "stderr": "Error from server (ServerTimeout): the server cannot complete the requested operation at this time, try again later (put imagestreams dotnet-runtime)", "stderr_lines": ["Error from server (ServerTimeout): the server cannot complete the requested operation at this time, try again later (put imagestreams dotnet-runtime)"], "stdout": "imagestream \"dotnet\" replaced", "stdout_lines": ["imagestream \"dotnet\" replaced"]}

PLAY RECAP **********************************************************************************************************************************************************************************
admin1                     : ok=30   changed=1    unreachable=0    failed=0   
localhost                  : ok=26   changed=0    unreachable=0    failed=0   
master1                    : ok=347  changed=85   unreachable=0    failed=1   
node1                      : ok=167  changed=33   unreachable=0    failed=0   
node2                      : ok=167  changed=33   unreachable=0    failed=0   



Failure summary:


  1. Hosts:    master1
     Play:     Upgrade default router and default registry
     Task:     Import Centos Image streams
     Message:  All items completed

```

##### Additional Information

* Vagrant box `centos/7`
* https://github.com/openshift/openshift-ansible-contrib 

Someone else in the *#openshift-users* channel on the Kubernetes community Slack server has also run into a similar issue were their Ansible run failed on this same exact step.

**Comment by vrutkovs** [comment]
That sounds like a temporary issue when images are being pulled. Does this occur every time?

**Comment by karthik101** [comment]
Is this issue resolved ?  i seem to get this in the latest 3.9 release.
